docs/additional_packaging.md

```diff
@@ -2,9 +2,22 @@
 
 To extend the build process, you can create a `additional_packaging.py` file in the same file level where you have your globalConfig file.
 
-This file should have the `additional_packaging` function, which accepts add-on name as its only argument.
+This file should at least have:
+
+- the `cleanup_output_files` function, which accepts `output_path` (str), `add-on name` (str) as its arguments.
```

Comment:
For the users it might be a little bit confusing what is the `output_path` here. When user decides to change output directory during the build process he would need to change it also in the additional_packaging. I wonder if we can make it more generic since we already have this parameter during the build process `_get_build_output_path`.

Reply:
I agree that it may be a bit confusing. What do you guys think about leaving only `additional_packaging` method and passing other arguments (like `output_path`), through e.g. env variables?

Reply:
I understand. My only concern is that when the user runs the command for example:
`ucc-gen build --output a/b/c`
he must also change the `output_path` to `a/b/c` in `additional packaging` and if he later run:
 `ucc-gen build --output x/y/z` he must remember to change it again the `additioanl packaging` to `x/y/z`

Reply:
Understood. I have addressed this issue in PR #1455.